.github/workflows/ci.yml

```diff
@@ -42,7 +42,9 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
+        with:
+          ref: 'main'
```

Comment:
According to https://github.com/actions/checkout#usage,
should we leave this empty instead of hardcoding it as main? What problem does it solve?

Reply:
Yesterday I seemingly needed it for CI to pass (it didn't make sense at the time, I just did it). But I just removed it, things seem to work today